data/human/news.txt

```diff
@@ -2380,6 +2386,9 @@ news "graffiti on spaceport wall"
 			"Better half a sentence, "
 			"To the moon, and not coming back."
 			"On a small space-station / early in the morning / seven spaceships landed / nicely in a line / and the captain / transmitted departure / tick tick woosh woosh / here we go"
-			"Time flies like an Arrow, fruit flies like a Banana."
+			"Time flies like an Arrow, fruit flies like a banana."
 			"The panda eats shoots and leaves. The hunter eats, shoots and leaves."
 			"The hyperlane to hell is paved with good intentions."
+			"Because I could not stop for Death - He kindly stopped for me - The Shuttle held but just one bunk - Most Peacefully. / We slowly flew; He knew no haste - And I had put away - My labor and my leisure too, - For His Civility." # Based on Emily Dickinsons "Because I could not stop for Death"
+			"We passed the Carrier, where Soldiers strove - In Fighters; in the Void - We passed the Bays with Docked Drones - We passed the Hardened Hull. / Or rather; It passed Us - The Dew drew quivering and Chill - For decompressed, my spacesuit - My tether, torn away." # Based on Emily Dickinsons "Because I could not stop for Death"
+			"We paused before a House that seemed - A Cylinder to be launched - The Roof was scarcely visible - The Flag; covering around. / Since then, 'tis Centuries, and yet - Feels shorter than the Day - I first surmised the Torpedo Bay - Aimed toward Eternity." # Based on Emily Dickinsons "Because I could not stop for Death"
```

Comment:
```suggestion
			"Because I could not stop for Death - He kindly stopped for me - The Shuttle held but just one bunk - Most Peacefully. / We slowly flew; He knew no haste - And I had put away - My labor and my leisure too, - For His Civility." # Based on Emily Dickinson's "Because I could not stop for Death"
			"We passed the Carrier, where Soldiers strove - In Fighters; in the Void - We passed the Bays with Docked Drones - We passed the Hardened Hull. / Or rather; It passed Us - The Dew drew quivering and Chill - For decompressed, my spacesuit - My tether, torn away." # Based on Emily Dickinson's "Because I could not stop for Death"
			"We paused before a House that seemed - A Cylinder to be launched - The Roof was scarcely visible - The Flag; covering around. / Since then, 'tis Centuries, and yet - Feels shorter than the Day - I first surmised the Torpedo Bay - Aimed toward Eternity." # Based on Emily Dickinson's "Because I could not stop for Death"
```